src/Microsoft.ML.Legacy/Runtime/EntryPoints/ModelOperations.cs

```diff
@@ -7,12 +7,16 @@
 using Microsoft.ML.Runtime.Data;
 using Microsoft.ML.Runtime.EntryPoints;
 using Microsoft.ML.Runtime.Learners;
+using System;
 using System.Linq;
 
+#pragma warning disable 612
 [assembly: LoadableClass(typeof(void), typeof(ModelOperations), null, typeof(SignatureEntryPointModule), "ModelOperations")]
+#pragma warning restore 612
 
 namespace Microsoft.ML.Runtime.EntryPoints
 {
+    [Obsolete]
     public static class ModelOperations
```

Comment:
>ModelOperations [](start = 24, length = 15)

This would be an example of a set of entry-points that does not reference the legacy namespaces/projects at all, and should have been borne out somewhere where it will be preserved. I am fairly certain that NimbusML uses this.